src/components/CheckEditor/CheckEditor.tsx

```diff
@@ -20,19 +9,43 @@ import { getDefaultValuesFromCheck, getCheckFromFormValues } from './checkFormTr
 import { validateJob, validateTarget } from 'validation';
 import CheckTarget from 'components/CheckTarget';
 import { Subheader } from 'components/Subheader';
+import { HorizonalCheckboxField } from 'components/HorizonalCheckboxField';
 import { CheckSettings } from './CheckSettings';
 import { ProbeOptions } from './ProbeOptions';
 import { CHECK_TYPE_OPTIONS, fallbackCheck } from 'components/constants';
 import { useForm, FormContext, Controller } from 'react-hook-form';
+import { GrafanaTheme } from '@grafana/data';
 
 interface Props {
   check?: Check;
   instance: SMDataSource;
   onReturn: (reload: boolean) => void;
 }
 
+const getStyles = (theme: GrafanaTheme) => ({
+  marginBotton: css`
+    margin-bottom: ${theme?.spacing.sm};
```

Comment:
No need for `?` operator here.

Reply:
Its funny, when using the`const styles = useStyles(getStyles)` , the theme passed to `getStyles` by the hook is typed as optional